tests/network/hotplug.go

```diff
@@ -405,6 +405,79 @@ var _ = SIGDescribe("nic-hotunplug", func() {
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
 	})
+
+	Context("a running VM", func() {
+		var vm *v1.VirtualMachine
+		var vmi *v1.VirtualMachineInstance
+
+		const guestNICName = "eth2"
+
+		BeforeEach(func() {
+			By("creating a NAD")
+			Expect(createBridgeNetworkAttachmentDefinition(
+				testsuite.GetTestNamespace(nil), networkName, linuxBridgeName)).To(Succeed())
+
+			By("running a VM")
+			opts := append(
+				libvmi.WithMasqueradeNetworking(),
+				libvmi.WithNetwork(libvmi.MultusNetwork(linuxBridgeNetworkName1, networkName)),
+				libvmi.WithNetwork(libvmi.MultusNetwork(linuxBridgeNetworkName2, networkName)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding(linuxBridgeNetworkName1)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding(linuxBridgeNetworkName2)),
+			)
+			vmi = libvmi.NewAlpineWithTestTooling(opts...)
+			vm = tests.NewRandomVirtualMachine(vmi, true)
+
+			var err error
+			vm, err = kubevirt.Client().VirtualMachine(testsuite.GetTestNamespace(nil)).Create(context.Background(), vm)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+			EventuallyWithOffset(1, func() error {
+				vmi, err = kubevirt.Client().VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &metav1.GetOptions{})
+				return err
+			}, 120*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+			libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
+
+			By("unplug interface")
+			ExpectWithOffset(1, libnet.InterfaceExists(vmi, guestNICName)).To(Succeed())
+			ExpectWithOffset(1, kubevirt.Client().VirtualMachine(vm.Namespace).RemoveInterface(context.Background(), vm.Name,
+				&v1.RemoveInterfaceOptions{Name: linuxBridgeNetworkName2},
+			)).To(Succeed())
+		})
+
+		DescribeTable("hot-unplug network interface succeed", func(plugMethod hotplugMethod) {
+			waitForUnplugIfaceApplyOnVMISpec(vmi, linuxBridgeNetworkName2)
+			vmi = verifyDynamicInterfaceChange(vmi, plugMethod)
+
+			Expect(libnet.InterfaceExists(vmi, guestNICName)).NotTo(Succeed(),
+				"unplugged interface should not exist in guest")
+			Expect(vmispec.LookupInterfaceStatusByName(vmi.Status.Interfaces, linuxBridgeNetworkName2)).To(BeNil(),
```

Comment:
I find this check redundant, `verifyDynamicInterfaceChange` already verifies it.